### PR TITLE
Added different wiring of dactyl

### PR DESCRIPTION
**Pull request by skakunm**
## Description
Added a new keyboard, it is similar to the Dactyl, but has 3x5 configuration with 5 thumb keys. Wiring is not compatible with the original dactyl since I used different wiring, so I had to make a different keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/skakunm_dactyl/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
```
Comment (drashna) [inline keyboards/handwired/skakunm_dactyl/keymaps/default/config.h:18]: None of this is needed, and can be removed.

File: keyboards/handwired/skakunm_dactyl/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/handwired/skakunm_dactyl/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/handwired/skakunm_dactyl/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+#define _LAYER0 0
+#define _LAYER1 1
+#define _LAYER2 2
```
Comment (drashna) [inline keyboards/handwired/skakunm_dactyl/keymaps/default/keymap.c:6]:
```suggestion
enum custom_layers {
    _LAYER0,
    _LAYER1,
    _LAYER2,
};
```

File: keyboards/handwired/skakunm_dactyl/skakunm_dactyl.c [hunk 4]
```diff
@@ -0,0 +1 @@
+#include "skakunm_dactyl.h"
```
Comment (drashna) [inline keyboards/handwired/skakunm_dactyl/skakunm_dactyl.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/handwired/skakunm_dactyl/skakunm_dactyl.h [hunk 5]
```diff
@@ -0,0 +1,31 @@
+#pragma once
```
Comment (drashna) [inline keyboards/handwired/skakunm_dactyl/skakunm_dactyl.h:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/handwired/skakunm_dactyl/skakunm_dactyl.h [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+#include "quantum.h"
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+    #include <avr/io.h>
+    #include <avr/interrupt.h>
+#endif
+#endif
```
Comment (drashna) [inline keyboards/handwired/skakunm_dactyl/skakunm_dactyl.h:11]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/skakunm_dactyl/keymaps/default/config.h [hunk 7]
```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 Marko Skakun <skakun.marko@gmail.com>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```
Comment (drashna) [inline keyboards/handwired/skakunm_dactyl/keymaps/default/config.h:3]: This entire file could be removed, as these are the default behaviors.

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]
This requires a [readme](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md)

**Review by keyboard-magpie — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**